source/getting-started/admin-onboarding-tasks.rst

```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
-      
-User Experience Optimizations
------------------------------
+Control product access with permissions
+----------------------------------------
 
-We highly recommend the following best practices, configuration options, and features for an optimal Mattermost user experience.
+You can customize which users can perform specific actions, such as creating teams, managing channels, posting reactions, `adding custom emojis <https://docs.mattermost.com/configure/configuration-settings.html#enable-custom-emoji>`__, or editing or deleting posts. See our `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation for details.
 
-**1. Upgrade your Mattermost server**
+Team creation
+-------------
 
-When you upgrade your Mattermost server frequently, your users can access new features, improved user experiences, bug fixes, security fixes, and Mobile App compatibility.
+Mattermost won’t limit you to the number of teams you can create; however, a public and an internal team are typically sufficient. See our `Creating Teams <https://docs.mattermost.com/messaging/creating-teams.html>`__ documentation for details.
 
-Mattermost releases regular updates to `Mattermost Team Edition <https://mattermost.com/>`_ and `Mattermost Enterprise Edition <https://mattermost.com/pricing-self-managed/>`_. The `Mattermost Changelog <https://docs.mattermost.com/install/self-managed-changelog.html>`_ provides all information about changes in each version.
+Streamline conversations with channels
+--------------------------------------
 
-Upgrading your Mattermost server only takes a few minutes. See the `Upgrade Guide <https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html>`__ for step-by-step instructions.
+1. Channels can be public and open, private and restrictive, direct with another user, direct with multiple users, or read-only. 
 
-**2. Install plugins**
+ - Convert channels from private to public and from public to private as needed. See our `Managing Channels <https://docs.mattermost.com/messaging/managing-channels.html>`__ documentation for details.
+ - Read-only channels are perfect for announcements because it’s easy to recall that information later. Update `channel moderation settings <https://docs.mattermost.com/onboard/advanced-permissions.html#read-only-channels-e20>`__ to set any channel as read-only.
 
-You can enable plugins and integrations to connect your team's workflows and toolsets into Mattermost. Plugins and integrations customize and extend the Mattermost platform.
+2. When creating any channel, we recommend using Markdown to populate the channel header with useful information and links relevant to all channel members, such as specifications, agendas, or other shared files. In addition, a soft channel naming convention helps users create and name new channels consistency, and find those channels easily later.
```

Comment:
What is a "soft channel naming convention"? I think this would be a good place to link to that naming convention page in the docs! :)

Reply:
Great point! I've elected to remove some of this content because it has introduced unnecessary duplication.